app/services/lambda_jobs/runner.rb

```diff
@@ -16,11 +18,21 @@ def run(&local_callback)
           payload: args.to_json,
         )
       else
-        job_class.handle(
-          event: args,
-          context: nil,
-          &local_callback
-        )
```

Comment:
I'm all in favor of a more realistic lambda setup in development, however, I think we may want to keep this block form for the time being. 

Today in production (where lambda is disabled), this inline block form is called. I think we want to keep that, because otherwise we would block the response back to the client on another request back to the IDP (in addition to the wait on the vendor).